docs/UserGuide.md

```diff
@@ -169,15 +189,27 @@ Examples:
 
 ### Adding a remark: `addremark`
 
-Adds a remark to an existing internship in the application.
+Adds a remark or modifies the existing one, of an to an existing internship at the specified `INDEX`.
 
-Format: `addremark INDEX [/remark REMARK]`
+<span style="color: #f66a0a;">Format:</span> `addremark INDEX [/remark REMARK]`
 
-* Adds a remark or modifies the existing one, of an existing internship at the specified `INDEX`. The index refers to the index number shown in the displayed internship list. The index **must be a positive integer** 1, 2, 3, …​
-* Existing remarks will be updated to the input values.
+* The index refers to the index number shown in the displayed internship list. The index **must be a positive integer** 1, 2, 3, …
+* The remark cannot be blank.
 
-Examples:
-*  `addremark 1 /remark This internship has a behavioural interview!` Adds or modifies the remark of the internship at index 1 to be `This internship has a behavioural interview!`.
+<span style="color: #f66a0a;">Examples:</span>
+*  `addremark 1 /remark This internship has a behavioural interview!` <br> Adds or modifies the remark of the internship at index 1 to be `This internship has a behavioural interview!`.
+ 
+<div markdown="span" class="alert alert-success">
+
+💡 **Try It Out:**<br>
+
+1. Input:<br>
+   `addremark 1 /remark This internship has a behavioural interview!`<br>
+   ![Before adding a remark](./images/add-remark/addremark-before.png)<br><br>
+
+2. Successful Output after adding a remark:<br>
+   ![After successfully adding a remark](./images/add-remark/addremark-after.png)
+</div>
```

Comment:
Should there be a common errors section as disccused in our meeting? eg blank remark, error output and solution to fix it (not be blank)

Reply:
Oh, I situated them in the format section currently